src/Resources/views/SyliusAdminBundle/Order/Label/State/fully_refunded.html.twig

```diff
@@ -0,0 +1,4 @@
+<span class="ui green label">
```

Comment:
IMO you should change the color of fully refunded state, because fulfilled is also green

Reply:
Purple has not been used so far... I guess. Changed!